siprachanh/TennisMatchLove
# Sc invite list

#### Changes Made
1. Added file `invite.js` to `invites` directory.
1. Modified file `InviteList.js` to include `rendering of invites list` .
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sc-InviteList
    ```
2. Open a new Terminal tab (⌘T or Ctrl+Shift+T) and navigate to the server directory.
3. Test app functionality.
    >This is a work in progress. There are error codes to reconcile.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > There are extraneous comments included at the moment.
